package.json

```diff
@@ -17,7 +17,9 @@
     "lerna": "^2.2.0",
     "mocha": "^3.4.2",
     "nyc": "^11.0.3",
-    "tslint": "^5.12.0"
+    "tslint": "^5.12.0",
```

Comment:
note: I follow the pattern we have in the main repo: an extension or example application should not dictate what linter / linter version or compiler/compiler version is used to develop. Rather these should be dictated by a given project's configuration. 

This is why `tslint` and `typescript` are defined in this project's root `package.json` and not in the extensions or example applications. This way if we use a the example app from this repo or one based on it, or a Theia application that contains e.g. @theia/cpp-debug, to develop in JS./TS project, we will not override their choices of compiler/linter.